### ISSUE TITLE
Does it support to Transformer models

### ISSUE THREAD

**Issue by youchangxin** [EasyFL-AI/EasyFL#9]
Is it possible to apply this FL architecture in transformer model?

**Comment by weimingwill** [comment]
Yes. It supports the transformer model. You just need to register the transformer model following this tutorial https://easyfl.readthedocs.io/en/latest/tutorials/model.html
